### DISCUSSION TITLE
Form Errors with array values & React-Select

### DISCUSSION THREAD

**Discussion by writes-of-spring** [react-hook-form/react-hook-form#6051] [Q&A] [answered]
With a React select multi select, using Controller and validation with at least one option selected, the formState errors types doesn't seem to match the expected result.

typescript error when trying to access errors.selectedOptions?.message
Property 'message' does not exist on type '(DeepMap<{ value: string; label: string; }, FieldError> | undefined)[]'

if I log out errors.selectedOptions I get {type: "validate", message: "", ref: Object} as expected.

CSB Link: https://codesandbox.io/s/practical-pare-9i6d9?file=/src/App.tsx

Thanks!

**Comment by bluebill1049** [comment, marked as answer]
https://react-hook-form.com/ts#NestedValue
